### PR TITLE
Feature/deadm 1353 airflow

### PR DESCRIPTION
**Pull request by tebadi**

### CODE REVIEW

File: dags/sentinel-2_data_transfer.py [hunk 1]
```diff
@@ -38,57 +43,167 @@ def extract_src_key(src_url):
...
+    data = json.loads(content)
+    return data
+
+def africa_extent():
+    """
+    Load Africa boundaries
+    :return: Shape object of the boundaries
+    """
+
+    boundaries = json.load(open("data/africa-extent.json"))
```
Comment (alexgleith) [inline dags/sentinel-2_data_transfer.py:72]: Do you need open as well?

File: dags/sentinel-2_data_transfer.py [hunk 2]
```diff
@@ -38,57 +43,167 @@ def extract_src_key(src_url):
...
+
+def scene_stac_utm(stac):
+    """
+    Extract the UTM zone from the STAC file
+    :return: UTM zone
+    """
+
+    return stac['properties']['proj:epsg']
+
+def reproject_scene_geom_to_epsg_4326(stac, scene_geometry):
```
Comment (alexgleith) [inline dags/sentinel-2_data_transfer.py:83]: I just found out about this library, which might be a better way to do this: https://pypi.org/project/mgrs/

It'll only handle the full scene footprint, but will work fine...

File: dags/sentinel-2_data_transfer.py [hunk 3]
```diff
@@ -38,57 +43,167 @@ def extract_src_key(src_url):
...
+
+    for rec in messages:
+        body = json.loads(rec)
+        message = json.loads(body['Message'])
+        scene_geometry = shape(message["geometry"])
+
+        # Extract URL of the json file
+        urls = [message["links"][0]["href"]]
+        is_in_africa_flag = is_in_africa(s3_hook, scene_geometry,
+                                                    africa_footprint, urls[0])
```
Comment (alexgleith) [inline dags/sentinel-2_data_transfer.py:135]: This indentation is a bit ugly... but not a big deal!

File: dags/sentinel-2_data_transfer.py (hunk 1 above)
Comment (tebadi) [reply dags/sentinel-2_data_transfer.py:72]: json.load needs a file object so, yes it needs the open.

File: dags/sentinel-2_data_transfer.py (hunk 2 above)
Comment (tebadi) [reply dags/sentinel-2_data_transfer.py:83]: I'll add this library as part of future improvements - I need to test it first.

File: dags/sentinel-2_data_transfer.py (hunk 3 above)
Comment (tebadi) [reply dags/sentinel-2_data_transfer.py:135]: Oh, it is ugly.  Will fix and push later.

**Pull request merged**